### DISCUSSION TITLE
Compatability with Obsidian Sync?

### DISCUSSION THREAD

**Discussion by vcavallo** [epwalsh/obsidian.nvim#652] [Q&A] [answered]
Hello

Looks like I've finally found the obsidian/zettelkasten nvim plugin I've been dreaming about! I'm just beginning to install it now and have this question:

I currently use Obsidian Sync (moved away from git repo syncing after installing Obsidian on far too many mobile devices). I am assuming that in order to have changes from the local filesystem picked up by Obsidian Sync, I'll need to pop open the Obsidian app on the machine.  
Are there are any plans for obsidian.nvim integration with Obsidian Sync? Is that even feasible, given whatever API (or lack thereof) and encryption they are using?

**Comment by epwalsh** [comment, marked as answer]
Hey @vcavallo this is not something I've looked into. You might find more answers in the official Obsidian forum :/
